src/com/spikes2212/genericsubsystems/commands/MonitoredMoveLimitedSubsystem.java

```diff
@@ -11,10 +11,28 @@
 	private LimitedSubsystem limitedSubsystem;
 	private Supplier<Double> speedSupplier;
 
+	/**
+	 * build a command that moves a certain limited subsystem in a certain speed
+	 * 
+	 * @param limitedSubsystem
+	 *            -the limited subsystem on which the command activates
```

Comment:
requires is better than activate, because it's consistant with the Susbsystem class